atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CQLKeyValueService.java

```diff
@@ -169,11 +169,11 @@ protected void initializeConnectionPool() {
         clusterBuilder.withCompression(Compression.LZ4);
 
         if (config.sslConfiguration().isPresent()) {
+            // We could switch to newer netty SSL configuration (perf++), but all of our other stuff
+            // uses the built-in java SSL configuration and doing something different could be a headache
             SSLContext sslContext = SslSocketFactories.createSslContext(config.sslConfiguration().get());
-            SSLOptions sslOptions = new SSLOptions(sslContext, SSLOptions.DEFAULT_SSL_CIPHER_SUITES);
+            JdkSSLOptions sslOptions = JdkSSLOptions.builder().withSSLContext(sslContext).build();
             clusterBuilder.withSSL(sslOptions);
-        } else if (config.ssl().isPresent() && config.ssl().get()) {
```

Comment:
no reason to support this deprecated SSL configuration in this else-if for a KVS that no one uses yet